BugFix: Classification Store Attribute Not Displaying Correct Value In Grid

torqdev opened the pull request:
When inheritance is enabled, the classification store attribute value is not being displayed correctly in the grid. 

Check to see if classification store attribute is an array before trying to access the `value` key. If not array, check to see if attribute has a value (similar to the logic when getting the inherited data: https://github.com/pimcore/admin-ui-classic-bundle/blob/1.7/src/Service/GridData/DataObject.php#L352)

torqdev committed:
Check to see if attribute value is an array before accessing via value.

torqdev committed:
Remove unneeded empty check

robertSt7 commented:
@torqdev I am not able to reproduce this. Could you please provide some more info like the type in the classification store you used? Thanks

torqdev commented:
Hi @robertSt7 - If inheritance is turned on, and the parent object has a value it does show on child objects as expected. But once the child value is overridden it only shows the inherited value in the grid.

Added classification store attributes to grid, and added values to parent attributes. Values are inherited to child accessory part. Editing the child accessory part continues to only display the parent inherited value.
![image](https://github.com/user-attachments/assets/5c58ffcb-0d55-4081-8ecf-0c37450f70db)

Opening up child accessory part shows correct new values after editing in grid.
![image](https://github.com/user-attachments/assets/50f091f5-c74a-49c9-9b65-81c71d58da48)

robertSt7 commented:
@torqdev Thanks for this fix
